packages/js-client-grpc/src/qdrant-client.ts

```diff
@@ -9,6 +11,7 @@ export type QdrantClientParams = {
     url?: string;
     host?: string;
     timeout?: number;
+    check_compatibility?: boolean;
```

Comment:
```suggestion
    checkCompatibility?: boolean;
```
Following the convention.